CHANGELOG.md

```diff
@@ -57,6 +57,11 @@ This is an example of a brief overview of the _Major_ or _Minor_ version changes
 - `lib/icons/raw/` will no longer exist in packages published v9 or later. For raw SVG
   icons, use `lib/icons/svg/`.
 
+#### Renamed InlineBlockList component
+- `InlineBlockList` component is now `SeparatorList` as we want the naming to have more
```

Comment:
(minor) we can explain that difference briefly

```
- `InlineBlockList` is now `SeparatorList`. This component is primarily used for configuring separator characters between items in an inline-block list. An upcoming component `GapList`, will render an inline-block list with configurable spacing between items.
```

Reply:
i'm not sure about this honestly; your call

Reply:
I feel like explaining the difference of a component that is yet to be released doesn't make sense. That being said, I think the fix here is just to release that component in v9, and I'm almost done with it.

Reply:
@akdetrick As previously discussed, I think moving a lot of things to v9 helps simplify things. GapList is now set to merge into v9, so I think adding this explanation now makes more sense. I took your explanation and modified it slightly, let me know how it works for you now.

Reply:
I'll merge this now, but we can easily make adjustments before v9 is merged.